query/organization/org_default_repo_permission_none.sql

```diff
@@ -2,10 +2,16 @@ select
   -- Required Columns
   html_url as resource,
   case
+    when default_repo_permission is null then 'skip'
```

Comment:
@jchrisfarris Should it be `skip` or `info`? I think `info` is better, as that's a way to tell the user they need to manually verify somehow. Please see https://github.com/turbot/steampipe-mod-github-compliance/blob/67c01dddf393d263006a0ce4b235458be8aa45e5/query/organization/org_two_factor_required.sql as a reference.

Reply:
If the user is not an owner, they cannot manually verify either. I chose skip because in these cases the user is not responsible for the GH Org.  

I can go with info is that is the convention.